packages/node_modules/@node-red/editor-client/src/js/keymap.json

```diff
@@ -17,6 +17,7 @@
         "ctrl-space": "core:toggle-sidebar",
         "ctrl-p": "core:toggle-palette",
         "ctrl-,": "core:show-user-settings",
+        "ctrl-alt-l": "core:clear-debug-messages",
```

Comment:
It should probably be worth noting that this doesn't actually the `ctrl-alt-l` shortcut by default to the workspace. I need to manually add it in the keyboard shortcuts.

This was unexpected as the `ctrl-g l` shortcut worked without intervention. I can only imagine something else is interfering with it being a globally scoped shortcut.